src/components/common/DomainSettingsSection.tsx

```diff
@@ -1,28 +1,31 @@
 import { makeStyles, Theme } from '@material-ui/core/styles';
 import * as React from 'react';
-import { Typography } from '@material-ui/core';
+import { IconButton, Typography } from '@material-ui/core';
```

Comment:
nit: Not sure if it's needed to be fixed, it looks like the space under Worflow/tasks # is visually bigger than above. It's a bit bothering me, but I'm not sure if reducing it - will make it look better. Fully up to you
<img width="619" alt="Screen Shot 2022-04-07 at 3 03 35 PM" src="https://user-images.githubusercontent.com/55718143/162331381-be090054-5e36-4ee4-997c-9e07891523bc.png">

Reply:
As I reused typography from other similar titles on the page and added the icon, bottom spacing may have accidentally increase, indeed! I will make sure it's equal top and bottom